src/test/java/com/hazelcast/aws/AwsClientTest.java

```diff
@@ -32,19 +32,6 @@
 public class AwsClientTest {
 
     @Test(expected = IllegalArgumentException.class)
-    public void testAwsClient_whenInvalidKey() {
-        AwsConfig awsConfig = new AwsConfig();
```

Comment:
when only access key is defined. If both is defined then we should throw InvalidConfigurationException. User must define only one configuration.